_templates/release.qmd

```diff
@@ -10,3 +10,20 @@ We are very excited to announce the release of a new {{ pkgname }} version {{ v
 Here is an automatically generated summary of the changes in this version.
 
 {{ notes }}
+
+## Acknowledgements
+
+```{r ctbs, results='asis', echo=FALSE, message = FALSE}
+releases <- gh::gh("GET /repos/{owner}/{repo}/releases", 
+               owner = "epiverse-trace",
+               repo = "{{ pkgname }}") |>
+  purrr::discard(~ grepl("\\d+\\.\\d+\\.[1-9]\\d*$", .x$tag_name))
+
+ctbs <- usethis::use_tidy_thanks(
+  paste0('epiverse-trace/', 
+    "{{ pkgname }}"), 
```

Comment:
```suggestion
  "epiverse-trace/{{ pkgname }}", 
```

Sorry for coming up with something new right at the end of the review but that's the last thing :grimacing:.

Explanation: `knitr::knit_expand()` has a mustache kind of approach and will do a search and replace in the code so no need to paste anything here.

Reply:
No need to apologize - I appreciate the input <3